lib/pushmi_pullyu/aip/creator.rb

```diff
@@ -1,65 +1,40 @@
-require 'archive/tar/minitar'
```

Comment:
`archive/tar/minitar` is apparently depreacted. Use `minitar` instead.

https://github.com/halostatue/minitar#description